test/OpenTelemetry.Exporter.Zipkin.Tests/ZipkinExporterTests.cs

```diff
@@ -177,6 +177,7 @@ public void ZipkinExporterIntegrationTest(bool useShortTraceIds)
         internal static Activity CreateTestActivity(
            bool setAttributes = true,
            Dictionary<string, object> additionalAttributes = null,
+           bool convertAttributeValuesToString = true,
```

Comment:
I introduced this to enable testing the code path in `ProcessTags` that handles `net.peer.port` when it is an integer. Meh? Better ideas?

My understanding is that this helper method does a `ToString` on all attribute values to eliminate any serialization differences between net452 and other target compilations. The net452 compilation uses Newtonsoft.Json and other compilations use the newer System.Text.Json. Should we be at all concerned that these two compilations result in serializing the Zipkin payload differently?

Reply:
@alanwest Are you saying if you put an int port into attributes, Zipkin sends it as an int? If yes, that might be a bug in the exporter. I think in Zipkin spec everything is sent as strings. /cc @eddynaka

Reply:
Yes, Zipkin will send it as an int for the net452 compilation that uses `Newtonsoft.Json`. If you remove the `ToString()` from here and run the Zipkin tests, you'll see what I mean:

https://github.com/open-telemetry/opentelemetry-dotnet/blob/c88120dc58b5b3647cf7aea42ffca157b754bfcc/test/OpenTelemetry.Exporter.Zipkin.Tests/ZipkinExporterTests.cs#L236

So, if the Zipkin spec requires strings then yea I think this is a bug.

Reply:
@alanwest, I think we should change to bet 452 as well. When I changed the others to output as string I didn't look at net452. Let me know if u will change or I can open another pr

Reply:
@eddynaka if you're interested in doing that PR, that would be super. I think what I will do for now is revert the changes to this helper method, but I think that if you land a fix for 452 then this `ToString()` should be able to be removed.

Reply:
just opened a pr solving this!